dali/operators/image/convolution/gaussian_blur_gpu/gaussian_blur_impl_f32.cu

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) 2020, NVIDIA CORPORATION. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "dali/operators/image/convolution/gaussian_blur_gpu.h"
+
+namespace dali {
+namespace gaussian_blur {
+
+template op_impl_uptr GetGaussianBlurGpuImpl<float, float>(const OpSpec&, DimDesc);
+
+}  // namespace gaussian_blur
+}  // namespace dali
```

Comment:
Maybe some comment to explain why these files were split would be useful?

Reply:
Added a comment explaining that for `GetGaussianBlurGpuImpl`.